doc/source/howtos/upgrade_guide.rst

```diff
@@ -136,98 +168,38 @@ To upgrade the cluster, follow these steps for each node:
-
-      To prevent any users from accessing this node while performing
-      work on it, you may add `--skip-networking
-      <https://dev.mysql.com/doc/refman/8.0/en/server-options.html#option_mysqld_skip-networking>`_
-      to the startup options and use a local socket to connect, or
-      alternatively you may want to divert any incoming traffic from
-      your application to other operational nodes.
+#. Now, start the cluster node with 8.0 packages installed, |pxc| will upgrade
+   the data directory as needed - either as part of the startup process or a
+   state transfer (IST/SST).
 
-#. When the upgrade is done, stop the ``mysqld`` process.
-   You can either run ``sudo kill`` on the ``mysqld`` process ID,
-   or ``sudo mysqladmin shutdown`` with the MySQL root user credentials.
-
-   .. note:: On CentOS, the :file:`my.cnf` configuration file
-      is renamed to :file:`my.cnf.rpmsave`.
-      Make sure to rename it back
-      before joining the upgraded node back to the cluster.
-
-#. Now you can join the upgraded node back to the cluster.
-
-   In most cases, starting the ``mysql`` service
-   should run the node with your previous configuration::
+   In most cases, starting the ``mysql`` service should run the node with your
+   previous configuration. For more information, see :ref:`add-node`.
 
    .. code-block:: bash
 
       $ sudo service mysql start
 
-   For more information, see :ref:`add-node`.
-
-   .. note:: As of version |version|,
-      |PXC| runs with :ref:`pxc-strict-mode` enabled by default.
-      This will deny any unsupported operations and may halt the server
-      upon encountering a failed validation.
-
-      If you are not sure, it is recommended to first start the node
-      with the :variable:`pxc_strict_mode` variable set to ``PERMISSIVE``
-      in the in the |MySQL| configuration file, :file:`my.cnf`.
-
-      After you check the log for any experimental or unsupported features
-      and fix any encountered incompatibilities,
-      you can set the variable back to ``ENFORCING`` at run time::
+   .. note::
 
-       mysql> SET pxc_strict_mode=ENFORCING;
+      On CentOS, the |file.centos-conf| configuration file is renamed to
+      :file:`my.cnf.rpmsave`.  Make sure to rename it back before
+      joining the upgraded node back to the cluster.
 
-      Also switch back to ``ENFORCING`` may be done by restarting the node
-      with updated :file:`my.cnf`.
+   |strict-mode| is enabled by default, which may result in denying any
+   unsupported operations and may halt the server. For more information, see
+   :ref:`upgrade-guide-changed-strict-mode`.
 
+   |opt.encrypt-cluster-traffic| is enabled by default. You need to configure
+   each node accordingly and avoid joining a cluster with unencrypted cluster
+   traffic. For more information, see
+   :ref:`upgrade-guide-changed-traffic-encryption`.
+      
 #. Repeat this procedure for the next node in the cluster
    until you upgrade all nodes.
 
```

Comment:
You can't have some nodes in the cluster that have the pxc_encrypt_cluster_traffic enabled and some disabled. As mentioned in the encryption docs, if you don't have the encryption enabled on 5.7 cluster, trying to join the cluster with pxc_encrypt_cluster_traffic=ON (even if you have the certificates set) would fail. To enable the encryption you need to re-bootstrap the cluster again.